soccer/planning/LineKickPlanner.cpp

```diff
@@ -0,0 +1,201 @@
+#include "LineKickPlanner.hpp"
```

Comment:
Oh sorry! I haven't had time to help on this stuff recently because of
internship madness. I'll look at those last files I changed tonight.

On Jul 23, 2016 1:22 PM, "Jay Kamat" <notifications@github.com> wrote:

> In soccer/planning/LineKickPlanner.cpp
> <https://github.com/RoboJackets/robocup-software/pull/767#discussion_r71977720>
> :
>
> > @@ -0,0 +1,201 @@
> > +#include "LineKickPlanner.hpp"
>
> I'm going to check this off because I'll assume its still in progress 😪
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RoboJackets/robocup-software/pull/767/files/822344ee363977ddc93ade8c8e6a72450a7791c6#r71977720>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB8XBu05qcst-_mWRNZ_D2N14jaN95PGks5qYnf-gaJpZM4JDqtT>
> .
>